requirements.txt

```diff
@@ -3,16 +3,19 @@ autograd==1.1.6
 backports-abc==0.4
 backports.shutil-get-terminal-size==1.0.0
 certifi==2016.8.31
+click==6.6
 configparser==3.5.0
 decorator==4.0.10
 entrypoints==0.2.2
 enum34==1.1.6
+Flask==0.11.1
 functools32==3.2.3.post2
 future==0.15.2
 ipykernel==4.5.0
 ipython==5.1.0
 ipython-genutils==0.1.0
 ipywidgets==5.2.2
+itsdangerous==0.24
```

Comment:
I think it comes with flask? It's just some signing library: http://pythonhosted.org/itsdangerous/